Assets/Scripts/GameOverManager.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+
+
+public class GameOverManager : MonoBehaviour
+{
+    GameObject[] racerList;
+    List<GameObject> finishedRacers;
+    // Start is called before the first frame update
+    void Start()
+    {
+        racerList = GameObject.FindGameObjectsWithTag("Racer");
+        finishedRacers = new List<GameObject>();
+        Debug.Log(racerList.Length);
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        if (finishedRacers.Count == racerList.Length)
+        {
+            SceneManager.LoadScene("GameOver");
```

Comment:
Do not load a whole new scene once the race is over. Prefer enabling a UI panel with the text on it. It will be much faster and it will be easier to pass the score of the racers to the UI.